server/plugin.go

```diff
@@ -98,7 +98,7 @@ func (p *Plugin) OnActivate() error {
 	// manifest store
 	mstore := p.store.Manifest
 	mstore.Configure(conf)
-	if conf.MattermostCloudMode {
+	if conf.MattermostCloudMode || conf.DeveloperMode {
```

Comment:
Actually, my initial comment was invalid. The marketplace install is only available in Cloud. Given that the store most likely won't work for devs, I'm inclined revert this change.
```suggestion
	if conf.MattermostCloudMode {
```

Reply:
Applying this changes as the current behavior also confused @DHaussermann